Makefile

```diff
@@ -128,7 +128,7 @@ uninstall: manifests kustomize
 
 .PHONY: deploy-cert-manager
 deploy-cert-manager:
-	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
+	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.6.0/cert-manager.yaml
```

Comment:
Hmmm, this change doesn't show up when I do `git diff origin/master Makefile`.  I think this might resolve itself when I squash before the merge happens, but I'm not certain.

Reply:
it is there: https://github.com/redhat-developer/service-binding-operator/blob/master/Makefile#L131

did you fetch the latest master from github?

Reply:
Apparently I hadn't.  It should be fixed now.